src/vg/civcraft/mc/bettershards/command/commands/GetShard.java

```diff
@@ -0,0 +1,60 @@
+package vg.civcraft.mc.bettershards.command.commands;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.ConsoleCommandSender;
+import org.bukkit.entity.Player;
+
+import net.md_5.bungee.api.ChatColor;
+import vg.civcraft.mc.civmodcore.command.PlayerCommand;
+import vg.civcraft.mc.mercury.MercuryAPI;
+import vg.civcraft.mc.mercury.PlayerDetails;
+
+public class GetShard extends PlayerCommand {
+
+	public GetShard(String name) {
+		super(name);
+		setIdentifier("shard");
+		setDescription("Gets the name of the shard you're in");
+		setUsage("/shard <name>");
+		setArguments(0, 1);
+	}
+	
+	@Override
+	public boolean execute(CommandSender sender, String[] args) {
+		if(args.length == 0) {
+			if(!(sender instanceof Player)) {
+				sender.sendMessage(ChatColor.RED + "Non-players need to specify a player to look up");
+				return true;
+			}
+			sender.sendMessage(ChatColor.GREEN + "You are currently in the shard '" + MercuryAPI.serverName() + "'");
+			return true;
+		} else {
+			if(!(sender.isOp() || (sender instanceof ConsoleCommandSender))) {
+				sender.sendMessage(ChatColor.RED + "You cannot look up the shard of another player!");
+				return true;
+			}
+			PlayerDetails details;
+			UUID player = UUID.fromString(args[0]);
```

Comment:
You need to surround this in try catch.  Listen for IllegalArgumentException and then set player = null.